ui/src/components/ripple-out/RippleOutContainer.js

```diff
@@ -39,6 +80,64 @@ const RippleOutContainer = () => {
                 CAMPAIGN
               </Text>
             </Flex>
+            <HStack
```

Comment:
<img width="301" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/24607458/296784ad-c4bd-42f4-8fb9-52602730d595">

please adjust text size for mobile